packages/account/src/Components/self-exclusion/self-exclusion-context.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
 
-const SelfExclusionContext = React.createContext();
+const SelfExclusionContext = React.createContext(null);
```

Comment:
as I see in the code, we use a lot of variables from SelfExclusionContext, I believe we should set types for this context